ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/manager/EthServerTest.java

```diff
@@ -55,7 +55,14 @@ public void setUp() {
         worldStateArchive,
         transactionPool,
         ethMessages,
-        new EthProtocolConfiguration(2, 2, 2, 2, 2, true));
+        EthProtocolConfiguration.builder()
+            .maxGetBlockBodies(2)
```

Comment:
Could we make a convenience method `allLimits(int)` that sets these values?  Same with `defaultConfig()` doing `builder().build()`?

Reply:
We could indeed.

Reply:
Done. And actually is it a nice way to show how to write custom setters in Builder